packages/grpc-google-iam-v1/README.rst

```diff
@@ -0,0 +1,108 @@
+Python Client for Cloud Identity and Access Management
+======================================================
+
+|stable| |pypi| |versions|
+
+`Cloud Identity and Access Management`_: 
+
+- `Client Library Documentation`_
+- `Product Documentation`_
+
+.. |stable| image:: https://img.shields.io/badge/support-stable-gold.svg
+   :target: https://github.com/googleapis/google-cloud-python/blob/main/README.rst#stability-levels
+.. |pypi| image:: https://img.shields.io/pypi/v/grpc-google-iam-v1.svg
+   :target: https://pypi.org/project/grpc-google-iam-v1/
+.. |versions| image:: https://img.shields.io/pypi/pyversions/grpc-google-iam-v1.svg
+   :target: https://pypi.org/project/grpc-google-iam-v1/
+.. _Cloud Identity and Access Management: https://cloud.google.com/iam/docs/
+.. _Client Library Documentation: https://cloud.google.com/python/docs/reference/grpc-iam/latest
+.. _Product Documentation:  https://cloud.google.com/iam/docs/
+
+Quick Start
+-----------
+
+In order to use this library, you first need to go through the following steps:
+
+1. `Select or create a Cloud Platform project.`_
+2. `Enable billing for your project.`_
+3. `Enable the Cloud Identity and Access Management.`_
+4. `Setup Authentication.`_
```

Comment:
In this and the other PRs, "Setup" should be two words: "Set up"

Reply:
This will be addressed in https://github.com/googleapis/synthtool/pull/2051